cli/paths.js

```diff
@@ -1,12 +1,19 @@
 import fs from 'fs'
+import os from 'os'
 import path from 'path'
 
+import packageJson from '../package.json'
+
 const appDirectory = process.cwd()
 const mouldDirectory = path.join(__dirname, '..')
+const editorDirectory = path.join(os.homedir(), '.mould')
 
-const resolveApp = (relativePath) => path.resolve(appDirectory, relativePath)
+const resolveApp = (relativePath) =>
+    path.resolve(appDirectory, relativePath)
 const resolveMould = (relativePath) =>
     path.resolve(mouldDirectory, relativePath)
+const resolveEditor = (relativePath) =>
+    path.resolve(editorDirectory, packageJson.version, relativePath)
```

Comment:
We store different Editor versions, because users might have different Mould versions among their projects

```
~
  /.mould
    /0.1.4
    /0.1.5
```